benchmarks/python/rope_ops.py

```diff
@@ -10,6 +10,16 @@
 
 from .model_configs import configs
 
+SEQ_LENGTHS = (
+    None,
```

Comment:
What does SEQ_LENGTH=None imply?

Reply:
That's just the case when we pull the `default` seq_length from the model config. But yeah these are already covered by our sweep.